clouddriver-google/src/test/groovy/com/netflix/spinnaker/clouddriver/google/deploy/ops/DeleteGoogleScalingPolicyAtomicOperationUnitSpec.groovy

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2015 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.google.deploy.ops
+
+import com.google.api.services.compute.Compute
+import com.netflix.spinnaker.clouddriver.data.task.Task
+import com.netflix.spinnaker.clouddriver.data.task.TaskRepository
+import com.netflix.spinnaker.clouddriver.google.deploy.description.DeleteGoogleScalingPolicyDescription
+import com.netflix.spinnaker.clouddriver.google.model.GoogleServerGroup
+import com.netflix.spinnaker.clouddriver.google.provider.view.GoogleClusterProvider
+import com.netflix.spinnaker.clouddriver.google.security.GoogleNamedAccountCredentials
+import spock.lang.Specification
+import spock.lang.Subject
+
+class DeleteGoogleScalingPolicyAtomicOperationUnitSpec extends Specification {
+  private static final SERVER_GROUP_NAME = "my-server-group"
+  private static final ACCOUNT_NAME = "my-account-name"
+  private static final REGION = "us-central1"
+  private static final PROJECT_NAME = "my-project"
+  private static final ZONE = "us-central1-f"
+
+  def setupSpec() {
+    TaskRepository.threadLocalTask.set(Mock(Task))
+  }
+
+  void "should delete zonal and regional autoscaling policy"() {
```

Comment:
Can use @Unroll here to report each iteration individually: http://spockframework.github.io/spock/javadoc/1.0/spock/lang/Unroll.html